Sources/NIOExtras/LengthFieldBasedFrameDecoder.swift

```diff
@@ -82,13 +109,23 @@ public final class LengthFieldBasedFrameDecoder: ByteToMessageDecoder {
     ///    - lengthFieldLength: The length of the field specifying the remaining length of the frame.
     ///    - lengthFieldEndianness: The endianness of the field specifying the remaining length of the frame.
     ///
-    public init(lengthFieldLength: ByteLength, lengthFieldEndianness: Endianness = .big) {
+    public convenience init(lengthFieldLength: ByteLength, lengthFieldEndianness: Endianness = .big) {
```

Comment:
@Lukasa do we want to deprecate this one or just leave as is? Doesn't really cause harm

Reply:
I'm fine with leaving it as-is.